server/app.js

```diff
@@ -1,122 +1,21 @@
 
-app.use(logger('dev'));
 
+const app = express();
+app.use(logger('dev'));
 app.use(bodyPaser.json());
-
 app.use(bodyPaser.urlencoded({ extended: false }));
 
+app.use('/api/users/', usersRouter);
+app.use('/api/recipes/', recipesRouter);
 
-// registration route
-app.post('/api/user/signup', userController.validateSignup);
-app.post('/api/user/signup', userController.checkEmailUniqueness);
-app.post('/api/user/signup', userController.checkUsernameUniqueness);
-app.post('/api/user/signup', userController.signUp);
-
-// login route
-app.post('/api/user/signin', userController.login);
-
-// create group route
-app.post('/api/group', auth.verifyToken);
-app.post('/api/group', groupController.validateGroupData);
-app.post('/api/group', groupController.createGroup);
-app.post('/api/group', groupController.getGroup);
-//  deleteGroup delete a group that was created before an error
-// app.post('/api/group', groupController.deleteGroup);
-
-
-// adding group member
-app.post('/api/group/:groupid/user', auth.verifyToken);
-app.post('/api/group/:groupid/user', userController.isGroupAdmin);
-app.post('/api/group/:groupid/user', userController.isUserValid);
-app.post('/api/group/:groupid/user', groupController.alreadyAMember);
-app.post('/api/group/:groupid/user', groupController.addMember);
-
-
-// post message to a group route
-app.post('/api/group/:groupid/message', auth.verifyToken);
-app.post('/api/group/:groupid/message', groupController.isValidGroup);
-app.post('/api/group/:groupid/message', groupController.isGroupMember);
-app.post('/api/group/:groupid/message', messageController.validateMessageData);
-app.post('/api/group/:groupid/message', messageController.createMessage);
-app.post('/api/group/:groupid/message', messageController.getMessage);
-
-// get all members of a  particular groups
-app.get('/api/group/:groupid/users', auth.verifyToken);
-app.get('/api/group/:groupid/users', userController.validateIds);
-app.get('/api/group/:groupid/users', groupController.isGroupMember);
-app.get('/api/group/:groupid/users', groupController.getGroupMembers);
-
-// get all admin of a  particular groups
-app.get('/api/group/:groupid/admin', auth.verifyToken);
-app.get('/api/group/:groupid/admin', userController.validateIds);
-app.get('/api/group/:groupid/admin', groupController.isGroupMember);
-app.get('/api/group/:groupid/admin', groupController.getGroupAdmin);
-
-// get all groups
-app.get('/api/groups', auth.verifyToken);
-app.get('/api/groups', groupController.getGroups);
-
-// get a group
-app.get('/api/group/:groupid', auth.verifyToken);
-app.get('/api/group/:groupid', groupController.validateGroupId);
-app.get('/api/group/:groupid', groupController.getGroup);
-
-
-// get all registered users
-app.get('/api/users', auth.verifyToken);
-app.get('/api/users', userController.getMembers);
-
-// get all group created by a user
-app.get('/api/user/admin/group', auth.verifyToken);
-app.get('/api/user/admin/group', userController.getCreatedGroups);
```

Comment:
Are you sure you didn't copy these codes from somewhere?

Reply:
Nope, you can check it out.  this is the link to my previous project https://github.com/codrex/PostIt